src/GameKit/GameKit.cs

```diff
@@ -130,23 +130,28 @@ public enum GKError : nint {
 		PlayerPhotoFailure = 26,  // iOS 8.0
 		UbiquityContainerUnavailable = 27, // iOS 8.0
 #endif
+		[iOS (10,0)][Mac (10,12)][TV (10,0)]
```

Comment:
It's not needed in this case, i.e. it's an `*Error` enum so it's an output (not a input).